src/widgets/openmediavault/OpenMediaVaultMemory.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Group, Paper, Progress, Text } from '@mantine/core';
+import { IconBrain } from '@tabler/icons-react';
+
+const OpenMediaVaultMemory = ({ info }: any) => {
+  const totalMemoryGB: any = (info.memTotal / 1024 ** 3).toFixed(2);
+  const freeMemoryGB: any = (info.memAvailable / 1024 ** 3).toFixed(2);
+  const usedMemoryGB: any = ((info.memTotal - info.memAvailable) / 1024 ** 3).toFixed(2);
+  const percentageUsed: any = ((usedMemoryGB / totalMemoryGB) * 100).toFixed(2);
+  const percentageFree: any = (100 - percentageUsed).toFixed(2);
```

Comment:
Why ``any``? All of these seem to be numerical to me

Reply:
The response from OMV is a string.